bindings/java/src/org/sleuthkit/datamodel/Blackboard.java

```diff
@@ -697,6 +697,9 @@ private boolean attributesMatch(Collection<BlackboardAttribute> fileAttributesLi
 	 * @throws TskCoreException exception thrown if a critical error occurs
 	 *                          within tsk core
 	 */
+	// TODO: this method needs to be removed.  
+	// The caller should instead use the newDataArtifact() api.
+	// 
```

Comment:
@jayaramcs had added this method and using it at the moment.  Brian's intent was to put a Todo as reminder  and we could replace it once things have stabilized.  

I think deprecating it now certainly makes sense. 

If all of Jay's changes are already in and this can be replaced right now, I don't mind doing it.